YIF_Backend/Controllers/InstitutionOfEducationAdminController.cs

```diff
@@ -105,5 +105,21 @@ public async Task<IActionResult> UpdateSpecialtyDescription([FromBody] Specialty
             var result = await _ioEAdminService.UpdateSpecialtyDescription(specialtyDescriptionUpdateApiModel);
             return Ok(result.Object);
         }
+
+        /// <summary>
+        /// Get all directions and specialties of admin
```

Comment:
stop sharmanka!!
and redo!!
replace logic of getting directions and specialties to IoEService/Controller, etc.

Reply:
Nu blen.